README.rst

```diff
@@ -193,9 +205,19 @@ work for this attribute, e.g. `MISRA`.
 chart
 -----
 
-This optional, second display option will draw a pie chart that visualizes the amount of results for each specified
-classification option. The parameters can be a list of classification options, comma-separated, or even
-plus-sign-separated to combine multiple options into the same slice.
+This optional, second display option will draw a pie chart that visualizes the amount of results for each allowed
+`<<attribute>>` option. Firstly, the attribute can be specified, followed by a colon. The default attribute is
+`classification`. Secondly, you have two optoins. Either you specify a list of attribue values, comma-separated or even
+plus-sign-separated for a merge into the same slice or else you define the minimum number of defects with the same
```

Comment:
Put comma here somewhere. `Minimum number` can also be `minimum threshold`

Reply:
fdae3884851cf7711b99f8e99c825935c7daabed